h2o-algos/src/main/java/hex/schemas/PCAV3.java

```diff
@@ -10,23 +10,24 @@
 public class PCAV3 extends ModelBuilderSchema<PCA,PCAV3,PCAV3.PCAParametersV3> {
 
   public static final class PCAParametersV3 extends ModelParametersSchemaV3<PCAParameters, PCAParametersV3> {
-    static public String[] fields = new String[] {
-      "model_id",
-      "training_frame",
-      "validation_frame",
-      "ignored_columns",
-      "ignore_const_cols",
-      "score_each_iteration",
-      "transform",
-      "pca_method",
-      "pca_impl",
-      "k",
-      "max_iterations",
-      "use_all_factor_levels",
-      "compute_metrics",
-      "impute_missing",
-      "seed",
-      "max_runtime_secs"
+    static public String[] fields = new String[]{
+        "model_id",
+        "training_frame",
+        "validation_frame",
+        "ignored_columns",
+        "ignore_const_cols",
+        "score_each_iteration",
+        "transform",
+        "pca_method",
+        "pca_impl",
+        "k",
+        "max_iterations",
+        "use_all_factor_levels",
+        "compute_metrics",
+        "impute_missing",
+        "seed",
+        "max_runtime_secs",
+        "export_checkpoints_dir"
```

Comment:
Unnecessary reformating makes git-blame useless. I try to keep the original formatting unless it is horrible ;)

Reply:
thats very subjective, I try to keep code in better shape when I touch it

Reply:
yep, it is subjective - no question about that

but it is objectively harder to figure out what happened with the code if there is a lot of unrelated refactorings ;)

I don't want to discourage you from improving - but I do suggest to make 2 PRs - one with refactorings and the second one with functional changes (not necessary here in the case of modified indentation in this list but certainly recommended for the automl changes). It is easy to hide an issue in a large PR (65 modified files).